README.md

```diff
@@ -63,14 +63,14 @@ All the built files are generated in the `dist` directory.
 - Go to `chrome://extensions`.
 - Turn ON `Developer mode`.
 - Click on `Load Unpacked`.
-- Select the `manifest.json` file in `dist` folder that was generated after the build step. The extension should now show up on the list.
+- Select the `dist` folder and it will check for `manifest.json` on its own. The extension should now show up on the list.
 - Go to any page and press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Space</kbd> or <kbd>Command</kbd> + <kbd>Shift</kbd> + <kbd>Space</kbd> (on Mac) to open UnTab.
 - NOTE: Whenever, the build files are changed, to get the changes synced with the installed extension, click on the reload icon next to the installed extension.
 
 #### Firefox
 - Go to `about:debugging` and select `This Firefox` on the left sidebar.
 - Click on `Load Temporary Add-on...` button.
-- Select the `manifest.json` file in `dist` folder that was generated after the build step. The extension should now show up on the list.
+- Select the `dist` folder and it will check for `manifest.json` on its own. The extension should now show up on the list.
```

Comment:
@tanishqvyas I'm not sure if Firefox checks it on its own, have you tested it?

Reply:
@tanishqvyas just checked, in the case of firefox while it automatically detects `manifest.json`, it requires the user to select some file in `dist` to trigger this behavior.

So, I feel let's keep this one unchanged as selecting `manifest.json` file directly seems more logical than selecting some random file and waiting for the browser to resolve correctly.